email-sending-concepts/one-click-unsubscribe.md

```diff
@@ -0,0 +1,130 @@
+
+## What is One-Click Unsubscribe?
+
+One-Click Unsubscribe provides email recipients with a standardized, **immediate method** to remove themselves from email lists or marketing communications. It appears either as a special link near the sender information in modern [email clients](/email-sending-concepts/smtp.md) (like Gmail's "Unsubscribe" button) or as a prominently displayed link in the email footer. When clicked, it immediately processes the unsubscribe request **without requiring additional steps** from the user.
+
+## How One-Click Unsubscribe Works
+
+The One-Click Unsubscribe mechanism is primarily implemented through standardized [email headers](/email-sending-concepts/email-headers) that work behind the scenes to facilitate the unsubscribe process. Email senders include special headers in their messages that email clients recognize and can present as unsubscribe options to users.
+
+The implementation centers around the **`List-Unsubscribe`** and **`List-Unsubscribe-Post`** headers, which provide recipient email clients with the necessary information to process unsubscribe requests automatically.
+
+| Header Type | Purpose | Example |
+|-------------|---------|---------|
+| **List-Unsubscribe (HTTP)** | Provides a URL endpoint for unsubscribe requests | `<https://example.com/unsubscribe?token=abc123>` |
+| **List-Unsubscribe (Mailto)** | Provides an email address for unsubscribe requests | `<mailto:unsubscribe@example.com?subject=unsubscribe>` |
+| **List-Unsubscribe-Post** | Enables true one-click functionality with secure POST requests | `List-Unsubscribe=One-Click` |
+
+Modern email clients like Gmail and Apple Mail scan these headers and display user-friendly unsubscribe options, typically as buttons or links near the sender information. When users click these options, the client handles the technical complexity of making the proper HTTP request or sending the unsubscribe email automatically.
+
+## Why is One-Click Unsubscribe Important?
+
+One-Click Unsubscribe has become a critical component of modern email marketing because it enhances [deliverability](/email-sending-concepts/deliverability) by gaining favor with email providers, reduces [spam complaints](/email-sending-concepts/complaints) by offering an easy alternative to marking emails as spam, improves user experience through a hassle-free opt-out method, manages list quality by filtering out uninterested subscribers, and simplifies **regulatory compliance** across multiple international jurisdictions. These benefits collectively result in better **[sender reputation](/email-sending-concepts/deliverability)**, higher engagement metrics, and a more efficient email marketing program.
```

Comment:
After this paragraph, you should include one that talks about when big inbox providers, such as gmail, yahoo, or outlook require one-click unsubscribe. (As far as I can remember, all of them require all senders to include one-click unsub if the sender sends more that 5000 emails....)

You should link those announcement articles from the sender websites.

Also, you should include it somewhere, that bluefox sends all non-transactional emails with one-clcik ubsub.